From af06ca9d46d46302c100f07e499d8aaa9ab32364 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 30 Jun 2020 15:58:38 +0000
Subject: [PATCH] IWYU: add includes for uint64_t and std::string

Bug: 819294
Change-Id: Iae844f8b6cf6bba879cb320c95ebf88f65e1385f
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2273161
Reviewed-by: Yao Xiao <yaoxia@chromium.org>
Commit-Queue: Yao Xiao <yaoxia@chromium.org>
Cr-Commit-Position: refs/heads/master@{#783978}
---
 components/federated_learning/sim_hash.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/components/federated_learning/sim_hash.h b/components/federated_learning/sim_hash.h
index 55030d9d2e3f..26087e74e9c7 100644
--- a/components/federated_learning/sim_hash.h
+++ b/components/federated_learning/sim_hash.h
@@ -5,7 +5,9 @@
 #ifndef COMPONENTS_FEDERATED_LEARNING_SIM_HASH_H_
 #define COMPONENTS_FEDERATED_LEARNING_SIM_HASH_H_
 
+#include <stdint.h>
 #include <set>
+#include <string>
 #include <unordered_set>
 
 namespace federated_learning {
-- 
2.26.2

